synphot/spectrum.py

```diff
@@ -778,6 +778,67 @@ def plot(self, wavelengths=None, **kwargs):  # pragma: no cover
         w, y = self._get_arrays(wavelengths)
         self._do_plot(w, y, **kwargs)
 
+    @classmethod
+    def from_Spectrum1D(cls, spec, keep_neg=False):
```

Comment:
To go along with my previous comment, this I guess would be 
```suggestion
    def from_spectrum1d(cls, spec, keep_neg=False):
```
or 
```suggestion
    def from_specutils(cls, spec, keep_neg=False):
```
?